README.md

```diff
@@ -84,17 +84,31 @@ Detailed Doxygen-genereated documentation is [available](https://codedocs.xyz/ey
 
 **Use involving CMake:**
 
-* Use CMake to configure, build and install the library. Then, in another CMake project, use `find_package(cuda_api_wrappers)` and make sure the library's install location is in CMake's package search path. This will let you use three targets within the `cuda-api-wrappers::` namespace: `runtime-and-driver`, `nvrtc` and `nvtx`.
-* Use CMake's `FetchContent` module to obtain the project source code and make it part of your own project's build, e.g.:
-   ```
-   include(FetchContent)
-   FetchContent_Declare(cuda-api-wrappers_library
-       GIT_REPOSITORY https://github.com/eyalroz/cuda-api-wrappers.git
-       GIT_TAG v12.34.56 # Replace this with a real available version
-   )
-   FetchContent_MakeAvailable(cuda-api-wrappers_library)
-   ```
-   The _same_ target names, with the namespaces, will be available in this case.
+***General CMAKE Usage***
+* After incorporating the library via either FetchContent or vcpkg, use find_package(cuda-api-wrappers) in your CMake project. Make sure the library's install location or build directory is in CMake's package search path.
```

Comment:
That's not exactly what I want to say... but I'll amend the REAMDE.md to mention the vcpkg availability.

Reply:
My mistake, I modified too much.
This PR is just a suggestion, 
If you have questions about vcpkg, I can help you as far as I know.